ci/generate.go

```diff
@@ -61,3 +66,33 @@ func (m *Generate) WebSdk() *dagger.Directory {
 		Directory("/work/client/web").
 		WithoutDirectory("node_modules")
 }
+
+func (m *Generate) Check(ctx context.Context) error {
+	app := goModuleCross("").
+		WithSource(m.Source).
+		Container().
+		WithEnvVariable("GOFLAGS", "-tags=musl")
+
+	result := app.
+		WithExec([]string{"go", "generate", "-x", "./..."}).
+		Directory("")
```

Comment:
nit: since you don't use `app` later, you could merge these

```suggestion
	result := goModuleCross("").
		WithSource(m.Source).
		WithEnvVariable("GOFLAGS", "-tags=musl")
		Exec([]string{"go", "generate", "-x", "./..."}).
		Directory("")
```